# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/cr145-rollup-as-rollup.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

From 354850338d29df5f9b538de29aae182c7298ce43 Mon Sep 17 00:00:00 2001
From: LN Liberda <lauren@selfisekai.rocks>
Date: Mon, 9 Feb 2026 05:08:49 +0100
Subject: [PATCH] Use rollup from rollup instead of @rollup/wasm-node

cr145-rollup-wasm.patch installs it as rollup and it's annoying to change
that, as either something has to be installed as rollup to satisfy peer
dependencies, or legacy-peer-deps=true has to be used. Both make diff
even bigger than already is.
---
 scripts/build/ninja/bundle.gni | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/devtools-frontend/src/scripts/build/ninja/bundle.gni b/third_party/devtools-frontend/src/scripts/build/ninja/bundle.gni
index 9b44b1f1b5..b11164a3e0 100644
--- a/third_party/devtools-frontend/src/scripts/build/ninja/bundle.gni
+++ b/third_party/devtools-frontend/src/scripts/build/ninja/bundle.gni
@@ -47,7 +47,7 @@ template("bundle") {
     }
   } else {
     node_action(target_name) {
-      script = "node_modules/@rollup/wasm-node/dist/bin/rollup"
+      script = "node_modules/rollup/dist/bin/rollup"
 
       forward_variables_from(invoker,
                              [
